src/py42/services/_connection.py

```diff
@@ -127,30 +127,35 @@ def from_device_connection(cls, connection, device_guid):
     def host_address(self):
         return self._get_host_address()
 
+    def get_login_configuration_for_user(self, username):
```

Comment:
Are we intentionally making this available to every type of Connection?
I don't think it would work on a microservice connection.

Would it be better to have a `LoginConfiguration` client that takes a simple `Connection` as a parameter.
Then, from the `sdk`, you would just do `sdk.login_configuration.get_for_user()` or however.

Reply:
We are also missing tests for this part (just the simple `_calls_get_with_expected_url_and_params()`, but if we separate this from `Connection`, those tests would be more isolated in their spot as well, which would be nice.

Reply:
Hmm, yeah that makes more sense.

Reply:
refactored it into a LoginConfigurationClient.